docs-site/docs/development/development_processes/code_review.md

```diff
@@ -0,0 +1,131 @@
+---
+sidebar_position: 1
+id: code_review
+title: Code Review Guidelines
+---
+
+This guide contains advice and best practices for performing code review, and having your code reviewed.
+
+All merge requests for IRIDA Next, whether written by a IRIDA team member or a wider community member, must go through a code review process to ensure the code is effective, understandable, maintainable, and secure.
+
+## Getting your pull request reviewed, approved, and merged
+
+Before you begin:
+* Familiarize yourself with the [contribution acceptance criteria](./pull_request_workflow/#contribution-acceptance-criteria)
+
+As soon as you have code to review, have the code **reviewed** by a reviewer. The reviewer can:
+* Give you a second opinion on the chosen solution and implementation.
+* Help look for bugs, logic problems, or uncovered edge cases.
+
+Getting your pull request **merged** also requires a maintainer. If it requires more than one approval, the last maintainer to review and approve merges it.
+
+### Acceptance checklist
+
+This checklist encourages the authors, reviewers, and maintainers of pull requests (MRs) to confirm changes were analyzed for high-impact risks to quality, performance, reliability, security, observability, and maintainability.
+
+Using checklists improves quality in software engineering. This checklist is a straightforward tool to support and bolster the skills of contributors to the IRIDA Next codebase.
+
+#### Quality
+
+1. You have self-reviewed this PR per [code review guidelines](./code_review).
+1. For the code that this change impacts, you beloeve that the automated tests validated functionality that is highly important to users.
```

Comment:
Typo in `believe` and add a comma after `you believe that`

Reply:
Addressed in https://github.com/phac-nml/irida-next/pull/105/commits/aa95bea4d612c311ca86792f706dd567be19251c

Leaving out comma change